keps/sig-instrumentation/distributed-tracing-kep.md

```diff
@@ -12,7 +12,7 @@ reviewers:
 approvers:
```

Comment:
name this something more specific, as we may do other distributed tracing in the k8s community in the future.